client/i18n/locales/english/intro.json

```diff
@@ -2057,46 +2107,82 @@
       "uglc": { "title": "199", "intro": [] },
       "vonu": { "title": "200", "intro": [] },
       "iejn": { "title": "201", "intro": [] },
-      "arrr": { "title": "202", "intro": [] },
+      "quiz-javascript-events": {
+        "title": "JavaScript Events Quiz",
+        "intro": ["Test what you've learned in this quiz on JavaScript Events."]
+      },
       "kaqq": { "title": "203", "intro": [] },
-      "ksfc": { "title": "204", "intro": [] },
-      "xeqa": { "title": "205", "intro": [] },
       "lab-random-background-color-changer": {
         "title": "Debug a Random Background Color Changer",
         "intro": [
           "For this lab, you will debug a random background color changer and fix the errors to make it work properly."
         ]
       },
+      "xeqa": { "title": "205", "intro": [] },
+      "quiz-debugging-javascript": {
+        "title": "Debugging JavaScript Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on Debugging JavaScript."
+        ]
+      },
       "dqth": { "title": "207", "intro": [] },
       "ilop": { "title": "208", "intro": [] },
       "qrer": { "title": "209", "intro": [] },
       "unyy": { "title": "210", "intro": [] },
       "svsl": { "title": "211", "intro": [] },
-      "wnmj": { "title": "212", "intro": [] },
+      "quiz-javascript-regular-expressions": {
+        "title": "JavaScript Regular Expressions Quiz",
+        "intro": [
+          "Test what you've learned in this quiz on JavaScript Regular Expressions."
+        ]
+      },
       "zalp": { "title": "213", "intro": [] },
       "hhvp": { "title": "214", "intro": [] },
       "egkv": { "title": "215", "intro": [] },
       "qlvw": { "title": "216", "intro": [] },
-      "cyrl": { "title": "217", "intro": [] },
+      "quiz-form-validation-with-javascript": {
+        "title": "Form Validation with Javascript Quiz",
```

Comment:
```suggestion
        "title": "Form Validation with JavaScript Quiz",
```

Reply:
✅ Fixed, will be added with next commit.